Android/app/src/main/java/uncommon/common/activity/MainActivity.java

```diff
@@ -8,22 +9,30 @@
 import com.facebook.AccessToken;
 import com.facebook.CallbackManager;
 import com.facebook.FacebookCallback;
-import com.facebook.FacebookException;
 import com.facebook.FacebookSdk;
-import com.facebook.login.LoginResult;
+import com.facebook.GraphRequest;
+import com.facebook.GraphResponse;
 import com.facebook.login.widget.LoginButton;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 import java.util.Arrays;
 
 import uncommon.common.R;
-
+import uncommon.common.utils.AccountManager;
+import uncommon.common.utils.FacebookLoginCallback;
 
 public class MainActivity extends AppCompatActivity {
 
-    private static final String EMAIL = "email";
+    private Context context = this;
     private static final String AUTH_TYPE = "rerequest";
+    private String userName;
+    private String userEmail;
 
+    private AccountManager accountManager = new AccountManager(context);
     private CallbackManager callbackManager;
+    private FacebookCallback mLoginCallback;

```

Comment:
No other member variables start with "m". Let's rename to loginCallBack (or facebookLoginCallback).